web/ui/react-app/src/ErrorAlert.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import { Alert } from 'reactstrap';
+
+interface ErrorProps {
+  summary: string;
+  message?: string;
+}
+
+const Error: FC<ErrorProps> = ({ summary, message }) => {
```

Comment:
`<Error>` is another case where I wonder if it's worth it to abstract it, but if we do, it would be good to change all the existing error messages to use this too.

Also, some of the summaries start with `Error fetching ...`, which stutters after we already say `Error:`. How about doing `<strong>{summary}</strong}: {message}`, and default `summary` to `Error:` when it's missing?

Reply:
I've removed the abstraction. I think you all are right that it's not necessary :)